make/harbor.yml

```diff
@@ -45,6 +45,9 @@ data_volume: /data
 #   # for more info about this configuration please refer https://docs.docker.com/registry/configuration/
 #   filesystem:
 #     maxthreads: 100
+#   # Uncomment following when you like configure disable 
```

Comment:
why there are two `#` ?

Reply:
Because there are two levels of comments.